serialization_spec/serialization.py

```diff
@@ -288,6 +288,14 @@ def expand_many2many_id_fields(model, serialization_spec):
                     expand_many2many_id_fields(many_related_models[key], each[key])
 
 
+def prefetch_queryset(queryset, serialization_spec, user=None, use_select_related=False):
```

Comment:
`use_select_related` should not have a default value: you should be forced to specify it